purchases/src/main/kotlin/com/revenuecat/purchases/models/PricingPhase.kt

```diff
@@ -54,14 +56,50 @@ data class PricingPhase(
             }
         }
 
+    /**
+     * Gives the price of the [PricingPhase] in the given locale in a weekly recurrence. This means that for example,
+     * if the period is monthly, the price will be divided by 4. It uses a currency formatter to format the price in
+     * the given locale. Note that this value may be an approximation.
+     * @param locale Locale to use for formatting the price. Default is the system default locale.
+     */
+    @JvmOverloads
+    fun pricePerWeek(locale: Locale = Locale.getDefault()): Price =
```

Comment:
I'm surprised codecov says these are not tested when we are adding tests in this PR 🤔. Not sure what's happening though... Any ideas?

Reply:
That's very weird. I'm investigating. 🔍

Reply:
I figured it out! I wasn't testing the overloads without `Locale`! https://github.com/RevenueCat/purchases-android/pull/1789/commits/46b8d7a94cdf67e504aa9dab39b9e49b37f44910

Reply:
Ohhh makes sense. Thanks for debugging that!